spyder/widgets/mixins.py

```diff
@@ -358,11 +358,11 @@ def get_current_word(self, completion=False):
             # (see below), the word to the left of the cursor won't be
             # selected), but only if the first character to the left is not a
             # white space too.
-            if not completion:
-                def is_space(move):
+            def is_space(move):
                     curs = self.textCursor()
                     curs.movePosition(move, QTextCursor.KeepAnchor)
                     return not to_text_string(curs.selectedText()).strip()

```

Comment:
These lines are now indented 8 spaces. Please make them to be indented just 4 spaces.